scripts/deblur

```diff
@@ -34,6 +34,9 @@ from deblur.workflow import (launch_workflow, trim_seqs, dereplicate_seqs,
 
 __version__ = "0.9.2.1"
 
+default_error_dist = '1,0.06,0.02,0.02,0.01,0.005,0.005' \
```

Comment:
What I'm concerned about is that this "default" is specified in multiple places, and there does not appear to be any mechanism to protect against a future developer making a change in one place and not another